### DISCUSSION TITLE
No Authentication Renders

### DISCUSSION THREAD

**Discussion by AnadyaNair** [supabase/supabase#17174] [Questions]
My supabase project neither does render new users anymore nor does it send emails for the magic link or mail confirmation for Auth.
What are the essentials that I must check?

**Comment by GaryAustin1** [comment]
Not sure what renders means in this context.

You could check the auth and postgres logs in the dashboard and see if there are any errors when you do your auth operation.
Pay particular attention to any trigger function you have on auth.users.  
Also if using Supabase SMTP you are limited to 4 emails per hour.
